unsupported/Eigen/CXX11/src/Tensor/TensorContractionGpu.h

```diff
@@ -581,7 +630,11 @@ EigenFloatContractionKernelInternal16x16(const LhsMapper lhs, const RhsMapper rh
     if (!CHECK_RHS_BOUNDARY) {
       if ((rhs_vert + 3) < k_size) {
         // just CHECK_RHS_BOUNDARY
+#if defined(EIGEN_HIPCC)
+	prefetch_rhs_hipcc(rhs_pf0, rhs_vert, rhs_horiz0)
```

Comment:
indentation

Reply:
do not think I follow you....am assuming you are talking about the "p" on line 634 not lining up with "/" on line 632? If so, that seems to be a git thing...they line up in both emacs and vi on my machine.

Reply:
then let's not worry about it now